src/main/java/pl/sda/javagda28/tourmanagingcrud/model/EventForm.java

```diff
@@ -1,9 +1,6 @@
 package pl.sda.javagda28.tourmanagingcrud.model;
 
-import lombok.AllArgsConstructor;
-import lombok.Builder;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Nie powinniśmy używać importów typu wildcard, [instrukcja jak to edytować](https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports/25916603)
A [tutaj informacja](https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad) dlaczego nie powinniśmy :)